clang/lib/CodeGen/TargetInfo.cpp

```diff
@@ -9301,6 +9316,11 @@ static bool requiresAMDGPUProtectedVisibility(const Decl *D,
            cast<VarDecl>(D)->getType()->isCUDADeviceBuiltinTextureType()));
 }
 
+void AMDGPUTargetCodeGenInfo::addAMDGCNMetadata(llvm::GlobalValue *GV,
```

Comment:
Nit pick. This wrapper seems unnecessary.

Reply:
I wanted to closely reflected the design of `NVPTXTargetCodeGenInfo`, which has `addNVVMMetadata`, removing that wrapper would introduce much more noise.